### ISSUE TITLE
关于数据集的来源

### ISSUE THREAD

**Issue by C-YEE00** [calubkk/RAAT#7]
请问你们可以提供数据集中每一条数据的来源吗？有没有标注这条数据是来自NQ或者WEBQ数据集的？

**Comment by calubkk** [comment]
您好，我们的训练集和测试集都是从其他开源数据集等比例随机采样获得的。我们并没有标注每条数据的source data。由于query并没有改写与重构。如果您需要source data，您可以尝试利用我们数据集中的query反向回溯原数据集（比如for循环检查NQ中是否有这条query）。

**Comment by C-YEE00** [comment]
我在原来的NQ数据集匹配你们的数据集，发现只有12条数据
我是将你们数据集和原数据集question中的句子变成set(),判断两个set是否相等判断的

**Comment by calubkk** [comment]
请问你匹配的是哪一个数据集？（train，test，dev？）

**Comment by C-YEE00** [comment]
是train，NQ-open.train.jsonl,在这个地址下载的https://github.com/google-research-datasets/natural-questions

**Comment by calubkk** [comment]
https://github.com/facebookresearch/DPR 可以用DPR的数据尝试一下。我们直接用DPR开源的NQ数据构造的。BTW，附加上论文中我们对于数据组成的介绍。

> The test set of RAG-Bench comprises 1000 randomly chosen samples from the test sets of three QA datasets, resulting in a total of 3000 samples.The training set consists of 1500 samples randomly selected from the training sets of the three datasets,totaling 4500 samples. The validation set, drawn from the training sets of three QA datasets, contains 300 samples. Notably, careful measures were taken to ensure no overlap with the training data of RAG-Bench.

**Comment by C-YEE00** [comment]
好的，我尝试一下，谢谢解答

**Comment by C-YEE00** [comment]
你好，我用dpr的数据可以匹配到trivia的1500条数据，但无法匹配nq的数据，请问你们可以提供webq的数据来源吗

**Comment by C-YEE00** [comment]
请你你们三个数据集分别是来源于DPR的哪一个文件，我发现dpr里每一个数据集的train也会对应多个文件

**Comment by C-YEE00** [comment]
目前我找到trivia的1500条数据，我在webquestions-train.json文件只找到1267条webq数据，而np数据集依然无法匹配到

**Comment by calubkk** [comment]
你好，非常感谢你的反馈。我们彻底检查了一下我们的开源数据集。webq的数据我们并没有检查出错误。但是在nq的匹配上确实存在着一定的问题。我们开源的是之前实验的错误版本。为了更好满足大家使用该数据集的需求，我们对其中的数据来源做了标注。这是新的数据集地址：https://drive.google.com/file/d/1u1XNg2Hk0vE8kJkogwXNNjbcDiiY50e1/view?usp=sharing

**Comment by calubkk** [comment]
欢迎继续关注我们的工作，如果有问题，请随时联系。

**Comment by C-YEE00** [comment]
好的，十分感谢

**Comment by JiananXie** [comment]
您好，可以请问关于三个数据集过滤后的subset是否可获取？十分感谢

**Comment by calubkk** [comment]
Hi，很抱歉，该subset我们并没有留下来保存。但是获取subset的方式很简单，您可以直接遍历三个数据集fullset，以是否拥有两条golden retrieval为标准进行筛选。nq和triviaqa带有检索的fullset可以在这里获取https://github.com/facebookresearch/DPR/blob/main/dpr/data/download_data.py      webq带有检索的fullset，dpr没有直接提供，如果您需要webq带有检索的fullset，可以邮件联系我。

**Comment by JiananXie** [comment]
> 目前我找到trivia的1500条数据，我在webquestions-train.json文件只找到1267条webq数据，而np数据集依然无法匹配到

请问，您是否有找到trivia test中的1000条数据呢？
